src/Components/Projects/Projects.jsx

```diff
@@ -0,0 +1,197 @@
+import React from "react";
+import facebookWebsite from './images/facebook-simulation-img.webp' 
+import coffeeWebsite from './images/coffee-website-img.webp' 
+import ECommerce from './images/e-commerce-img.webp' 
+import picShots from './images/pic-shots-img.webp' 
+import './projects.css';
+
+function Projects() {
+  return (
+    <section id="projects" className="projects">
+      <div className="container">
+        <h2>My Projects</h2>
+        <div className="project">
```

Comment:
since you are using the same format for rendering your projects, you can create an array of the projects
with this type
`interface Project {
title: string;
technologies: string[];
description: string;
githubLink: string;
img: string
}
`
and render them one by one